packages/core/helper-plugin/src/components/GenericInput.jsx

```diff
@@ -249,7 +249,11 @@ const GenericInput = ({
           name={name}
           onChange={(date) => {
             onChange({
-              target: { name, value: formatISO(date, { representation: 'date' }), type },
+              target: {
+                name,
+                value: date ? formatISO(date, { representation: 'date' }) : null,
```

Comment:
What do you think about adding a test? 🙏🏼

Reply:
done ✅